chipsec/modules/common/uefi/access_uefispec.py

```diff
@@ -178,10 +178,11 @@ def check_vars(self, do_modify):
                             self.logger.log_bad("Variable {} should be read only.".format(name))
                             res = ModuleResult.FAILED
 
-        self.logger.log('')
-        self.logger.log_bad('Variables with attributes that differ from UEFI spec:')
-        for name in uefispec_concern:
-            self.logger.log('    {}'.format(name))
+        if len(uefispec_concern) > 0:
```

Comment:
would it be better to use:
`if uefispec_concern:`

Reply:
👍